.php_cs.dist

```diff
@@ -10,8 +10,8 @@ return PhpCsFixer\Config::create()
     ->setRules([
         '@Symfony' => true,
         '@Symfony:risky' => false,
-        'phpdoc_align' => false,
-        'no_superfluous_phpdoc_tags' => false,
+        'phpdoc_align' => true  ,
+        'no_superfluous_phpdoc_tags' => true,
```

Comment:
Why the align? It looks not good.

Reply:
I think it almost make the code readable... please switch back

Reply:
### Symfony
```
    /**
     * Transforms the input given as first argument.
     *
     * @param bool|string $dummy   Some argument description
     * @param array       $options An options collection to be used within the transformation
     *
     * @return string|null The transformed input
     *
     * @throws \RuntimeException When an invalid option is provided
     */
    private function transformText($dummy, array $options = [])
    {
    ...
    }
```
See https://symfony.com/doc/current/contributing/code/standards.html

Reply:
> I think it almost make the code readable...

You're right - it almost does make the code readable 😄

Reply:
On a more serious note. It's possible that we can rector to understand "our" style but we should ask ourselves why we deviate from the established standards.